src/components/courselayout.tsx

```diff
@@ -0,0 +1,14 @@
+const CourseLayout = () => {
+  return (
+    <div className="bg-transparent-80 p-auto font-Malayalam mx-auto flex w-[95vw] flex-col items-center rounded-[1vw] bg-devstation-green-200 p-[2.5vw] text-center font-normal">
+      <div className="text-[4vw]">Our Projects</div>
+
+      <div className="text-[1.15vw]">
```

Comment:
```suggestion
      <div className="text-lg">
```
this is technically not header text, just the above was